### ISSUE TITLE
The keyword "directive " is lost

### ISSUE THREAD

**Issue by lipigirl** [nicolasdao/graphql-s2s#10]
If the keyword `directive` is used in the schema ( see [Schema directives](https://www.apollographql.com/docs/graphql-tools/schema-directives.html) ), it is lost after using the function transpileSchema.

**Comment by nicolasdao** [comment]
Hi @lipigirl,

This should be fixed with the latest release (i.e., 0.16.0). 

Let me know if that works on your side.

Thanks a lot for pointing that out. This really helps to make this project better.

Cheers,

Nic

**Comment by lipigirl** [comment]
Thanks a lot,the keyword `directive` worked,

But when the generic `<T>` is used together, there's another problem,

For example,

```JavaScript
 var schema =
directive @rest
 (
   type: String
   url: String
 ) on FIELD_DEFINITION | ENUM_VALUE
 
 type Manager<T> {
    name: String
 }
 
 type PageList<T> {
    results: [T]
 }
 
 type Query {
   querySome: PageList<Manager> @rest(type: "GET",url:"/198.168.0.1")
 }
```

```JavaScript
 var schema_output =
directive @rest
 (
   type: String
   url: String
 ) on FIELD_DEFINITION | ENUM_VALUE
 
 type Manager<T> {
    name: String
 }
 
 type PageList<T> {
    results: [T]
 }
 
 type Query {
   querySome: PageList<Manager>//lost @rest(type: "GET",url:"/198.168.0.1")
 }
```

**Comment by lipigirl** [comment]
@nicolasdao

**Comment by nicolasdao** [comment]
Hi @lipigirl ,

Damn, I'm sorry that this issue is not fully resolved. Thank you so much reporting that. I'll fix that asap.

**Comment by nicolasdao** [comment]
Hi @lipigirl ,

This issue should be fixed with the latest version (0.16.1). Can you give it a shot and let me know if that's working on your side?

**Comment by lipigirl** [comment]
Hi @nicolasdao 
This issue is perfect fixed,Thanks a lot~
